src/rime_api.cc

```diff
@@ -980,95 +980,141 @@ void RimeSetCaretPos(RimeSessionId session_id, size_t caret_pos) {
   return ctx->set_caret_pos(caret_pos);
 }
 
-RIME_API RimeApi* rime_get_api() {
```

Comment:
Ignore the indentation change.

Only added a new function to the API at the end of the `RimeApi` init block.